app/controllers/transactions_controller.rb

```diff
@@ -42,6 +44,25 @@ def destroy
     @transaction.destroy
   end
 
+  def copy
+    if @transaction.transfer?
+      @transaction = @transaction.dup
```

Comment:
@dissident I'm not sure if it's correct way... it will probably copy `created_at` and `updated_at` too? I think we need to copy only specific fields, like Category, Customer, Amount, Bank Account, Comment, Date.